fog/view/server/src/bin/router.rs

```diff
@@ -38,7 +43,7 @@ fn main() {
     );
 
     // TODO: Remove and get from a config.
-    let mut fog_view_store_grpc_clients = Vec::new();
+    let mut fog_view_store_grpc_clients = HashMap::new();
     let grpc_env = Arc::new(
         grpcio::EnvBuilder::new()
             .name_prefix("Main-RPC".to_string())
```

Comment:
:wrench: Perhaps this is in the scope of the TODO and not sure if people may look at the threads to differentiate which shards were _original_ and which ones were later added.  
These initial shards are created with the `Main-RPC` thread prefix and the new ones are added with `add-shard`.  
It probably depends if the thread names are used for logging and health, but it may be good to use similar names

Reply:
I don't think we'll need to differentiate the threads- they should be treated the same. If we need to differentiate shards then we can do so with ResponderId.